react.md

```diff
@@ -5,39 +5,43 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Correctable True/False: Mark the following TRUE or FALSE. If FALSE, correct the statement to be TRUE.
 
-- React is a modern, efficient answer to complex UI applications (true/false)
-- React will only render on the server using Node.js (true/false)
-- React is a full stack framework for modern web applications (true/false)
-- React is a flexible library that plays the role of V in an MVC framework (true/false)
-- You should always update a component's state directly using this.state (true/false)
-- React is made up of encapsulated components that manage their own state (true/false)
-- React components render HTML (true/false)
+- React is a modern, efficient answer to complex UI applications (TRUE/false)
+- React will only render on the server using Node.js (TRUE/false)
+- React is a front end framework for modern web applications (TRUE/false)
+- React is a flexible library that plays part of the role of V in an MVC framework (TRUE/false)
+- One of numerous ways to update a component's state directly, is using this.state (TRUE/false)
```

Comment:
When updating a component's state we want to use the method this.setState rather than reassigning this.state which is called updating state directly.